src/app/routes/cpsAsset/utils/getAdditionalPageData.test.js

```diff
@@ -126,19 +126,14 @@ describe('getAdditionalPageData', () => {
 
   describe('Optimizely Experiments', () => {
     describe('004_brasil_recommendations_experiment', () => {
-      it('should recommendations data from camino, unirecs content and unirecs hybrid engine', async () => {
+      it('should get recommendations data from unirecs content and unirecs hybrid engine', async () => {
         const expectedOutput = {
-          recommendations: recommendationsJson,
           datalabContentRecommendations: recommendationsJson,
           datalabHybridRecommendations: recommendationsJson,
         };
 
         hasRecommendations.mockImplementationOnce(() => true);
 
-        fetchMock.mock(
-          'http://mock-recommendations-path/recommendations/portuguese/brasil-59057279?Engine=unirecs_camino',
-          recommendationsJson,
-        );
```

Comment:
Just in line with the above comment, may be worth putting the `recommendations` object and `fetchMock` back in for this (with the `Engine=unirecs_camino` changed to `Engine=unirecs_datalab`